guides/getting-started/set-up/indy-vdr.md

```diff
@@ -0,0 +1,174 @@
+
+Currently, there are few limitations to using Indy VDR.
+
+- React Native projects leveraging [Expo](https://expo.dev) are not supported at the moment. Support for Expo will be added soon.
+- React Native Android support starts from 11 right now. Work is being done on support for lower versions.
+- When running in Node.JS, only Node.JS 18 is suported for now. See [Supported Node.JS versions for Indy VDR](#supported-nodejs-versions-for-indy-vdr)
+
+:::
+
+### Installing Indy VDR
+
+When using Aries Framework JavaScript with Indy VDR, there are a few extra dependencies that need to be installed. We need to install the `@hyperledger/indy-vdr`, which contains all the functionality to register objects on an Hyperledger Indy VDR. Secondly, we need to add native bindings for the specific platform `@hyperledger/indy-vdr-<platform>`. Currently there are bindings for Node.JS, as `@hyperledger/indy-vdr-nodejs`, and React Native as `@hyperlegder/indy-vdr-react-native`.
+
+<!--tabs-->
+
+# Node.JS
+
+```console
+yarn add @aries-framework/indy-vdr @hyperledger/indy-vdr-nodejs
+```
+
+# React Native
+
+```console
+yarn add @aries-framework/indy-vdr @hyperledger/indy-vdr-react-native
+```
+
+<!--/tabs-->
+
+### Configuration
+
+As you can see below, the Indy VDR module takes the native bindings and a list of networks. This list of networks will be used to resolve and register objects on.
+
+```typescript title="example"
+import { indyVdr } from '@hyperledger/indy-vdr-nodejs'
+import { IndyVdrModule } from '@aries-framework/indy-vdr'
+
+indyVdr: new IndyVdrModule({
+  indyVdr,
+  networks: [
+    {
+      indyNamespace: 'bcovrin:test',
+      isProduction: false,
+      genesisPath: './path/to/genesis.txn',
+      connectOnStartup: true,
+    },
+  ],
+})
+```
+
+#### indyVdr
+
+**Type**: `IndyVdr`
+
+the `indyVdr` key takes a class that implements all the native bindings for Indy VDR. This can be extracted from the `@hyperledger/indy-vdr-nodejs` package or the `@hyperledger/indy-vdr-react-native` package.
+
+#### networks
+
+**Type**: `IndyVdrPoolConfig[]`
+
+An array of indy networks to connect to. The list can contain the following object and it must include _either_ the [`genesisPath`](.md#indyledgersgenesispath) _or_ [`genesisTransactions`](#indyledgersgenesistransactions). It is important to know that the first ledger in the list ledgers will be used for registering the schema, credential definition, etc.
+
+##### indyNamespace
+
+**Type**: `string`
+
+The Indy namespace aka the name identifying the name of the network connecting to. See also [indy did method identifiers](https://hyperledger.github.io/indy-did-method/#indy-did-method-identifiers)
+
+##### `isProduction`
+
+**Type**: `boolean`
+
+Whether the ledger is a production ledger. This is used for the pick-up algorithm as production ledgers have priority.
```

Comment:
```suggestion
Whether the ledger is a production ledger. This is used for detecting which ledger to use in case of unqualified identifiers as production ledgers have priority.
```